src/services/completions.ts

```diff
@@ -54,7 +59,7 @@ namespace ts.Completions {
         }
 
         // Add keywords if this is not a member completion list
-        if (!isMemberCompletion && !isJsDocTagName) {
+        if (!isMemberCompletion && !(requestJsDocTag  && requestJsDocTagName)) {

```

Comment:
should not this be `!isMemberCompletion && !requestJsDocTag  && !requestJsDocTagName`?